src/BaseApiAbstract.php

```diff
@@ -462,6 +464,20 @@ function ($a) {
      */
     protected function sendRequest($uri, array $options, $method, $returnRawResponseBody = false)
     {
+        $config = $this->getHttpClient()->getConfig();
+        $debug = $config["debug"];
+        $handlerStack = $config["handler"];
+
+        if ($debug) {
+            $handlerStack->push(
+                Middleware::mapRequest(function(RequestInterface $request) {
+                    echo var_export((string) $request->getBody(), true) . PHP_EOL;
```

Comment:
I'm assuming this is not an error and will get buffered somewhere

Reply:
You mean dumping to stdout?